zyte_common_items/base.py

```diff
@@ -118,3 +115,45 @@ def _apply_field_types_to_sub_fields(cls, item: Optional[Dict]):
             item.update({key: cls.from_list(item.get(key, [])) for key, cls in (from_list or {}).items()})
 
         return item
+
+
+@_export_attrs()
+class AdditionalProperty:
+    name: str
+    value: str
+
+
+@_export_attrs(kw_only=True)
+class AggregateRating:
+    best_rating: Optional[float] = None
+    rating_value: Optional[float] = None
+    review_count: Optional[int] = None
+
+
+@_export_attrs()
+class Brand:
+    name: str
+
+
+@_export_attrs()
+class Breadcrumb:
+    name: str
+    link: str
+
+
+@_export_attrs()
+class Gtin:
+    type: str
+    value: str
+
+
+@_export_attrs(kw_only=True)
+class Image:
+    data_url: Optional[str] = None
+    url: Optional[str] = None
+
+
+@_export_attrs(kw_only=True)
+class Metadata:
+    date_downloaded: str
```

Comment:
```suggestion
    dateDownloaded: str
```

Reply:
No, it's the opposite :) it may be required by the guidelines, but likely it won't be available in the API for some time.